MAUI/Barcode/customization.md

```diff
@@ -0,0 +1,364 @@
+
+For one dimensional barcode, if this property is not set, the size of the smallest bar line is determined depending on the width available.
+
+The following code snippet shows the one dimensional barcode with `Module` property,
+
+{% tabs %}
+
+{% highlight xaml hl_lines="7" %}
+
+            <barcode:SfBarcodeGenerator Value="123456789" 
+                                        HeightRequest="150"
+                                        WidthRequest="240" 
+                                        ShowText="True" 
+                                        BackgroundColor="LightCyan">
+                <barcode:SfBarcodeGenerator.Symbology>
+                    <barcode:Codabar Module="1"/>
+                </barcode:SfBarcodeGenerator.Symbology>
+            </barcode:SfBarcodeGenerator>
+
+{% endhighlight %}
+
+{% highlight C# hl_lines="7" %}
+
+        SfBarcodeGenerator barcode = new SfBarcodeGenerator();
+        barcode.HeightRequest = 150;
+        barcode.WidthRequest = 240;
+        barcode.Value = "123456789";
+        barcode.ShowText = true;
+        barcode.BackgroundColor = Colors.LightCyan;
+        barcode.Symbology = new Codabar() { Module = 1 };
+        this.Content = barcode;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![.NET MAUI Barcode Generator Barcode Width Customization](images/customization/maui-barcode-width-customization.png)
+
+N> In the image above, the smallest bar line width is 1 logical pixel.
+
+Below code snippet shows the one dimensional barcode without the `Module` property,
+
+{% tabs %}
+
+{% highlight xaml %}
+
+        <barcode:SfBarcodeGenerator Value="123456789" 
+                                    HeightRequest="150"
+                                    WidthRequest="240" 
+                                    ShowText="True" 
+                                    BackgroundColor="LightCyan" Margin="50,20">
+            <barcode:SfBarcodeGenerator.Symbology>
+                <barcode:Codabar />
+            </barcode:SfBarcodeGenerator.Symbology>
+        </barcode:SfBarcodeGenerator>
+
+{% endhighlight %}
+
+{% highlight C# %}
+
+        SfBarcodeGenerator barcode = new SfBarcodeGenerator();
+        barcode.HeightRequest = 150;
+        barcode.WidthRequest = 240;
+        barcode.Value = "123456789";
+        barcode.ShowText = true;
+        barcode.BackgroundColor = Colors.LightCyan;
+        barcode.Symbology = new Codabar();
+        this.Content = barcode;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![.NET MAUI Barcode Generator Barcode Without Module](images/customization/maui-barcode-without-module.png)
+
+N>  In the image above, the smallest bar line width is calculated on the basis of the available width as a result of the total computed inputs(0’s and 1’s) divided by the available width.
+
+For two dimensional barcode , if the `Module` property is not set, the size of smallest dot is calculated based on the minimum of available width or height.
```

Comment:
If the `Module` property is not set for two-dimensional barcode, the size of smallest dot is calculated based on the minimum of available width or height.

Reply:
Changes updated.